src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -222,6 +222,44 @@ export const BucketLocation = requesterPaysWrapper({ onDismiss: _.noop })(({ wor
   return h(TooltipCell, [flag, ' ', regionDescription]);
 });
 
+export const AzureStorageDetails = ({ azureContext, storageDetails }) => {
+  return h(Fragment, [
+    h(InfoRow, { title: 'Cloud Name' }, [h(AzureLogo, { title: 'Microsoft Azure', role: 'img', style: { height: 16 } })]),
```

Comment:
The code did not change-- I just pulled it out into a component for testability.